jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/BasicFiltersViewImpl.html

```diff
@@ -0,0 +1,23 @@
+<div data-field="content" class="manage" style="height: 100%;">
+    <div class="col-md-12 tab-content kie-tab-content--filters" style="height: 100%;">
+        <form role="form" style="margin-top: 20px;">
+            <h5 class="kie-dock__heading--section" data-field="refine"></h5>
+            <div class="form-group">
+                <select class="selectpicker form-control" data-width="100%" data-field="refine-options"></select>
+            </div>
+            <div class="form-group">
+                <div class="input-group" data-field="filters-input">
+                    <span class="input-group-btn">
+                        <button class="btn btn-link" data-field="filters-input-help" type="button" data-toggle="popover" data-trigger="hover" data-html="true" title="" data-placement="bottom" data-container="body">
+                            <i class="fa fa-info-circle"></i>
```

Comment:
@cristianonicolai There is something different in the css applied, related with the 'info-circle'. In this view appears a border around the button when is clicked...

Reply:
@nmirasch ok, I will take a look at that.